dhis-2/dhis-support/dhis-support-test/pom.xml

```diff
@@ -77,6 +77,12 @@
       <artifactId>postgresql </artifactId>
       <version>1.10.1</version>
     </dependency>
+    <dependency>
+      <groupId>io.github.benas</groupId>
+      <artifactId>random-beans</artifactId>
+      <version>3.7.0</version>
```

Comment:
Put the dependency version in the root pom.xml file under the <dependencyManagement> section, then just include here without versioning. That way we ensure we only get one version of each lib in the system.

Reply:
Ok, so there is a bit of a problem with transitive dependencies in Maven. You can read here: https://maven.apache.org/plugins/maven-jar-plugin/examples/create-test-jar.html 
To solve the problem I have placed the `random-beans` lib in the main pom **without** the `test` scope. This solves the transitive dependency problem: if a sub-projects needs this lib it can declare it in the pom and use the `test` scope.